README.md

```diff
@@ -68,12 +68,14 @@ Request Body example:
                 'username': 'username',
                 'password': 'password',
                 'endpoint': 'url',
+                'auth': 'Kerberos',
                 'kernelname': 'pysparkkernel'
         }
         ```
 
-*Note that the kernelname parameter is optional and defaults to the one specified on the config file or pysparkkernel if not on the config file.*
+*Note: The auth can be either None, Basic_Access or Kerberos based on the authentication enabled in livy. kernelname parameter is optional and defaults to the one specified on the config file or pysparkkernel if not on the config file.*
```

Comment:
Actually, a full section on Kerberos auth is needed, as mentioned in a previous comment. Please add the note you have below regarding ticket management to that section, and explain that it is simply using the requests-kerberos package. Basically, give an overview of what type of kerberos authentications are supported vs which ones are not so that users know what to expect and how to configure the system.